apps/test-ultrasonic-driver/src/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include <cstdio>
+
+#include "UltraSonicSensor.h"
+
+// set up Ultrasonic sensor
+UltraSonicSensor sensor(PA1, PA2);
```

Comment:
This should not be assigned to arbitrary pins. maybe we should designate test pins in the pinmap? @younesr1 ?